.travis.yml

```diff
@@ -26,19 +26,28 @@ before_script:
   - |
       if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
       pip install git+https://github.com/SUPERAndroidAnalyzer/travis-cargo.git@fix-coveralls --user &&
-      export PATH=$HOME/.local/bin:$PATH
+      export PATH=$HOME/.local/bin:$PATH &&
+      (travis-cargo install rustfmt || true) &&
+      export PATH=$PATH:~/.cargo/bin &&
```

Comment:
I don't think this export is needed. In any case, that last `&&` breaks the script.